src/field/field255.rs

```diff
@@ -436,20 +431,9 @@ mod tests {
         assert_matches!(
             Field255::try_from_bytes(
                 &[
-                    0x7f, 0xff, 0xff, 0xff, 0xff, 0xff, 0xfe, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff,
+                    0xec, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff,
                     0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff,
-                    0xff, 0xff, 0xff, 0xff, 0xff, 0xee,
```

Comment:
To be honest I"m not sure why this was supposed to ever pass. As far as I can tell the value is larger than the modulus?

Reply:
The deleted test had an 0xfe n one of the higher words, so it wasn't more than the modulus.

Reply:
Ohhh missed that :) Added the test back.